info.go

```diff
@@ -508,6 +518,34 @@ func (pi *ProgramInfo) VerifiedInstructions() (uint32, bool) {
 	return pi.verifiedInstructions, pi.verifiedInstructions > 0
 }
 
+// KsymAddrs returns the ksym addresses of bpf programs, including subprograms,
+// of current bpf program. The addresses are same as the ones in /proc/kallsyms.
+//
+// Available from 4.18.
+//
+// The bool return value indicates whether this optional field is available.
+func (pi *ProgramInfo) KsymAddrs() ([]uint64, bool) {
```

Comment:
This should return `[]uintptr`. (makes it even easier to feed into MultiKprobe!)